eclipse/launchers/HeadlessGameServer_local_test.launch

```diff
@@ -8,7 +8,7 @@
 <listEntry value="1"/>
 </listAttribute>
 <stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="games.strategy.engine.framework.headlessGameServer.HeadlessGameServer"/>
-<stringAttribute key="org.eclipse.jdt.launching.PROGRAM_ARGUMENTS" value="triplea.game.host.console=true &#13;&#10;triplea.game= triplea.server=true &#13;&#10;triplea.port=3300&#13;&#10;triplea.lobby.host=127.0.0.1&#13;&#10;triplea.lobby.port=3300&#13;&#10;triplea.name=Bot1_TestServer &#13;&#10;triplea.lobby.game.hostedBy=Bot1_TestServer&#13;&#10;triplea.lobby.game.supportEmail=developer@gmail.com&#13;&#10;triplea.lobby.game.comments=automated_host"/>
```

Comment:
There's a subtle change on this line that is causing the Server Options dialog to be displayed when you run this launcher.

This set of parameters (note the space between the `=` and `triplea.server`):

```
triplea.game= triplea.server=true
```

was changed to

```
-Ptriplea.game=triplea.server=true
```

whereas it should be

```
-Ptriplea.game= -Ptriplea.server=true
```

Reply:
Good catch!